### DISCUSSION TITLE
Parameters tab is gone after reinstall Argocd v1.2.1

### DISCUSSION THREAD

**Discussion by benzsuankularb** [argoproj/argo-cd#19437] [Q&A] [answered]
My parameters tab is gone after delete and reinstalling Argocd v1.2.1.
What are the possibilities of misconfiguration for this?
Or where did it move to?

**Comment by crenshaw-dev** [comment]
Is the version actually v1.2.1? If so, the one and only task to be prioritized is to upgrade to a more recent version. 1.2.1 was released almost five years ago.

**Reply by benzsuankularb** [reply, marked as answer]
OK, I'm sorry for my ignorance.
The version is v2.12.0.
And the reason the parameters tab does not show up is because it is hiding under the "sources" tab.
I guess argocd has detected I'm using multiple git source.
